README.md

```diff
@@ -406,10 +406,11 @@ To enable automatic credentials rotation, you need to set the following paramete
 
 **The `credentialsRotationPolicy` is evaluated and executed during the [control loop](https://kubernetes.io/docs/concepts/architecture/controller/) which runs on every update or during
   a full reconciliation process. This means that the actual rotation time may potentially be greater than the specified duration in 'rotationFrequency'.**
```

Comment:
"greater than the specified duration" sound intimidating, need to specify times... e.g., "up to 10 hours"